spec/spec.md

```diff
@@ -82,9 +82,60 @@ A description to the reader on how the document is structured.
 
 ## Terminology
 
-Common terms in this document come from the specifications listed in (#reference).
+[[def:Authorization Request]]
+~ OAuth 2.0 Authorization Request extended by [[ref:OIDC]] and [[ref:OpenID4VP]]
 
-Note: the term Verifier in the [[ref: VC Data Model]] corresponds to a term Relying Party (RP) in [[ref:SIOPv2]].
+[[def:Authorization Response]]
+~ OAuth 2.0 Authorization Response extended by [[ref:OIDC]] and [[ref:OpenID4VP]]
+
+[[def:Claim, Claims]]
+~ An assertion made about a subject
+
+[[def:Decentralized Identifier, DID]]
+~ An identifier that enables verifiable, decentralized digital identity as defined by the W3C specification
```

Comment:
should be `identifier`. DIDs are not identity - it's just a way to get pubKeys by reference

Reply:
for the purpose of this section I am more comfortable with clarifying that the core property of DIDs is the ability to enable Clients to obtain key material and other metadata by reference.